### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by TheDancingClown**
### Admin verdict
- changes label for notes when editing verdict
Screenshot 2021-09-08 at 20 38 39

- adds grey background to saved verdict
Screenshot 2021-09-08 at 22 33 33

### Admin comments
- displays author name instead of email
- moves caption to below author
- adds red colour to delete link
- adds govuk-body style to comments
- adds grey background
Screenshot 2021-09-08 at 22 24 09
Screenshot 2021-09-08 at 23 03 25

### Assessor verdicts
- adds grey background
Screenshot 2021-09-08 at 22 33 47

### CODE REVIEW

File: app/assets/stylesheets/admin/page-applications.scss [hunk 1]
```diff
@@ -282,8 +282,8 @@
 }
 
 .form-container {
-  padding: 10px;
-  background: $grey-lighter;
+  padding: 20px;
+  background: #f3f2f1;
```
Comment (mauricioac) [inline app/assets/stylesheets/admin/page-applications.scss:286]: where is this used?
I dont think we should have these grey containers anymore.
If we do, we need to check for color contrast on all elements inside it.

File: app/views/admin/form_answers/_verdict.html.slim [hunk 2]
```diff
@@ -19,7 +19,7 @@
                       html: { class: "final-verdict-togglable" } do |f|
       .toggable-form aria-expanded=verdict.errors.any?.to_s
         .toggable-form__read
-          .govuk-body.toggable-form__content
+          .govuk-body.toggable-form__content.form-container
```
Comment (mauricioac) [inline app/views/admin/form_answers/_verdict.html.slim:22]: please check for color contrast again all elements

File: app/views/admin/form_answers/_comment_read_only.html.slim [hunk 3]
```diff
@@ -1,7 +1,9 @@
-.comment class="#{'comment-flagged' if comment_read_only.flagged?}"
-  .comment-header-m
+.comment.form-container class="#{'comment-flagged' if comment_read_only.flagged?}"
```
Comment (mauricioac) [inline app/views/admin/form_answers/_comment_read_only.html.slim:1]: please check for color contrast against all elements inside this

File: app/views/admin/form_answers/appraisal_form_components/_rag_section.html.slim [hunk 4]
```diff
@@ -1,7 +1,7 @@
 - form = AppraisalForm.evaluation_options_for(f.object, section)
 - editable = policy(f.object).update?
 
-.govuk-form-group.rag-section class="#{'form-edit' if f.object.public_send(section.desc).blank? && editable} form-#{section.label.parameterize}"
+.govuk-form-group.rag-section.form-container class="#{'form-edit' if f.object.public_send(section.desc).blank? && editable} form-#{section.label.parameterize}"
```
Comment (mauricioac) [inline app/views/admin/form_answers/appraisal_form_components/_rag_section.html.slim:4]: same here

File: app/views/admin/form_answers/appraisal_form_components/_verdict_section.html.slim [hunk 5]
```diff
@@ -1,7 +1,7 @@
 - form = AppraisalForm.verdict_options_for(f.object, section)
 - editable = policy(f.object).update?
 
-.govuk-form-group.verdict-section class="#{'form-edit' if f.object.public_send(section.desc).blank? && editable} form-#{section.label.parameterize}"
+.govuk-form-group.verdict-section.form-container class="#{'form-edit' if f.object.public_send(section.desc).blank? && editable} form-#{section.label.parameterize}"
```
Comment (mauricioac) [inline app/views/admin/form_answers/appraisal_form_components/_verdict_section.html.slim:4]: and here

File: app/assets/stylesheets/admin/page-applications.scss (hunk 1 above)
Comment (TheDancingClown) [reply app/assets/stylesheets/admin/page-applications.scss:286]: Used for admin comments and assessor appraisals. Passes AA standard

**Pull request merged**